ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -79,6 +79,83 @@
 #     )
 
 
+# def _size_strategy():
+#     return st.one_of(
+#         st.integers(),
+#         st.tuples(st.integers()),
+#         st.lists(st.integers(), min_size=3, max_size=3),
+#     )
+
+# def _scale_factor_strategy():
+#     return st.one_of(
+#         st.floats(),
+#         st.tuples(st.floats()),
+#         st.lists(st.floats(), min_size=3, max_size=3),
+#     )
```

Comment:
Removing the bounds from the `dtype_and_x` part is good but I think here they were quite useful. In `x`, below, bounding the values doesn't really change much, like the function can still run with unbounded values so it's probably best to leave them without bounds, but here I think you need them.

For example, if either of these come up with a negative value, I think the function can't handle that, and also if they come up with very large values the function can easily run out of memory (because it'll try to create a tensor with `3e+38` entries)